### PR TITLE
Update containerd to v1.3.1, runc v1.0.0-rc9

### PR DESCRIPTION
**Pull request by tao12345666333**
Signed-off-by: Jintao Zhang <zhangjintao9020@gmail.com>

**- What I did**

Update to runc v1.0.0-rc9
Full diff https://github.com/opencontainers/runc/compare/3e425f80a8c931f88e6d94a8c831b9d5aa481657...v1.0.0-rc9

Update to containerd  v1.3.1

Full diff https://github.com/containerd/containerd/compare/v1.3.0...v1.3.1

### CODE REVIEW

**Review by thaJeztah — APPROVED** [review]
LGTM

but we likely want to wait merging until this is in a containerd release and we updated to that version

**Comment by tao12345666333** [comment]
Thanks for your reply @thaJeztah 
I will update this PR when containerd released.

**Comment by AkihiroSuda** [comment]
@dmcgowan Do we plan to release v1.3.1 soon?

**Comment by thaJeztah** [comment]
@tao12345666333, looks like this needs a minor rebase 😓

**Comment by tao12345666333** [comment]
@thaJeztah Thanks for your ping. Addressed.

**Comment by thaJeztah** [comment]
Thanks! Discussing with containerd maintainers to tag a new release to unblock this PR 👍

**Comment by tao12345666333** [comment]
Flaky test: TestRestartContainerwithRestartPolicy  
xref: https://github.com/moby/moby/issues/32219

```
[2019-10-28T19:15:15.279Z]     --- FAIL: TestDockerSuite/TestRestartContainerwithRestartPolicy (13.77s)
[2019-10-28T19:15:15.279Z]         cli.go:29: assertion failed: 
[2019-10-28T19:15:15.279Z]             Command:  d:\CI\PR-40048\2\binary\docker.exe restart 3879b12310f171aa561e0a38f94ac9e0b63f11a1a749f2a5cb8526954ceec5f9
[2019-10-28T19:15:15.279Z]             ExitCode: 1
[2019-10-28T19:15:15.279Z]             Error:    exit status 1
[2019-10-28T19:15:15.279Z]             Stdout:   
[2019-10-28T19:15:15.279Z]             Stderr:   Error response from daemon: Cannot restart container 3879b12310f171aa561e0a38f94ac9e0b63f11a1a749f2a5cb8526954ceec5f9: Container 3879b12310f171aa561e0a38f94ac9e0b63f11a1a749f2a5cb8526954ceec5f9 is not running
[2019-10-28T19:15:15.279Z]             
[2019-10-28T19:15:15.279Z]             
[2019-10-28T19:15:15.279Z]             Failures:
[2019-10-28T19:15:15.279Z]             ExitCode was 1 expected 0
[2019-10-28T19:15:15.279Z]             Expected no error
```



https://ci.docker.com/public/blue/rest/organizations/jenkins/pipelines/moby/branches/PR-40048/runs/2/nodes/53/steps/216/log/?start=0

**Comment by tao12345666333** [comment]
https://github.com/containerd/containerd/releases/tag/v1.3.1

containerd v1.3.1 has been released, I will update this PR today.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Comment by thaJeztah** [comment]
ping @dmcgowan @crosbymichael @cpuguy83 PTAL

**Review by cpuguy83 — COMMENTED** [review]
LGTM

**Pull request merged**
